### DISCUSSION TITLE
Jiabaida/JBD BMS - RS232 und RS485 Parallelbetrieb

### DISCUSSION THREAD

**Discussion by NemoN** [shining-man/bsc_fw#23] [General]
Hallo zusammen,

ist es möglich das Jiabaida BMS gleichzeitig via RS485 (Victron VenusOS) + RS232 (BSC) zu verwenden?

Grüße
NemoN

**Comment by shining-man** [comment]
Ich kann es nicht sicher sagen, aber ich denke nicht, dass ein Parallelbetrieb von RS485 und RS232 geht. Du kannst es natürlich versuchen, dann wissen wir es sicher.
Was geht ist ein Parallelbetrieb des RS485 und BT. Soviel ich weiß hängt das BT-Modul bei dem JBD auch an einer seriellen Schnittstelle. Das könntest zu abziehen (ist nur gesteckt) und du hättest eine zweite Schnitstelle mit der du zum VenusOS/BSC kannst. Vorausgesetzt die Schnittstelle hat das gleiche Kommunikationsprotokoll. Natürlich hast du bei der Variante dann kein Bluetooth mehr.

Alternativ das JBD mit RS485/RS232 ans das BSC und von dort per CAN zum VenusOS.
